doc/nrf/releases_and_maturity/releases/release-notes-2.5.1.rst

```diff
@@ -0,0 +1,126 @@
+The SDK includes open source projects (TF-M, MCUboot, OpenThread, Matter, and the Zephyr RTOS), which are continuously integrated and redistributed with the SDK.
+
+Release notes might refer to "experimental" support for features, which indicates that the feature is incomplete in functionality or verification, and can be expected to change in future releases.
+To learn more, see :ref:`software_maturity`.
+
+Highlights
+**********
+
+This patch release adds changes on top of the :ref:`nRF Connect SDK v2.5.0 <ncs_release_notes_250>`.
+
+.. To do
+
+See :ref:`ncs_release_notes_251_changelog` for the complete list of changes.
+
+Release tag
+***********
+
+The release tag for the |NCS| manifest repository (|ncs_repo|) is **v2.5.1**.
+Check the :file:`west.yml` file for the corresponding tags in the project repositories.
+
+To use this release, check out the tag in the manifest repository and run ``west update``.
+See :ref:`cloning_the_repositories` and :ref:`gs_updating_repos_examples` for more information.
+
+For information on the included repositories and revisions, see `Repositories and revisions for v2.5.1`_.
+
+IDE and tool support
+********************
+
+`nRF Connect for Visual Studio Code extension <nRF Connect for Visual Studio Code_>`_ is the only officially supported IDE for |NCS| v2.5.1.
+
+:ref:`gs_app_tcm`, used to install the |NCS| automatically from `nRF Connect for Desktop`_, is available for Windows, Linux, and macOS.
+
+Supported modem firmware
+************************
+
+See `Modem firmware compatibility matrix`_ for an overview of which modem firmware versions have been tested with this version of the |NCS|.
+
+Use the latest version of the nRF Programmer app of `nRF Connect for Desktop`_ to update the modem firmware.
+See :ref:`nrf9160_gs_updating_fw_modem` for instructions.
+
+Modem-related libraries and versions
+====================================
+
+.. list-table:: Modem-related libraries and versions
+   :widths: 15 10
+   :header-rows: 1
+
+   * - Library name
+     - Version information
+   * - Modem library
+     - `Changelog <Modem library changelog for v2.5.1_>`_
+   * - LwM2M carrier library
+     - `Changelog <LwM2M carrier library changelog for v2.5.1_>`_
+
+Known issues
+************
+
+Known issues are only tracked for the latest official release.
+See `known issues for nRF Connect SDK v2.5.1`_ for the list of issues valid for the latest release.
+
+.. _ncs_release_notes_251_changelog:
+
+Changelog
+*********
+
+The following sections provide detailed lists of changes by component.
+
+Protocols
+=========
+
+This section provides detailed lists of changes by :ref:`protocol <protocols>`.
+
+Wi-Fi
+-----
+
+* Added improvements and fixes related to the following:
+
+  * :term:`Delivery Traffic Indication Message (DTIM)` power save functionality
+  * Firmware issues due to AP power cycle
+  * MAC address configuration
+  * General system stability
+
```

Comment:
Due to other changes, we had to adopted also some things on Matter, I would add those: 

Matter:
 - Fixed an issue with commissioning Matter over Wi-Fi devices to 5th fabric
 - Aligned the default values of attributes in ZAP files to comply with the Matter 1.1 specification
 - Fixed an issue with the Matter Bridge not being operational in some commercial ecosystems

Reply:
The last one is on PR: https://github.com/nrfconnect/sdk-nrf/pull/13399
But hopefully can be merged today :)